ivy/functional/frontends/jax/func_wrapper.py

```diff
@@ -88,3 +89,51 @@ def new_fn(*args, **kwargs):
 
 def to_ivy_arrays_and_back(fn: Callable) -> Callable:
     return outputs_to_frontend_arrays(inputs_to_ivy_arrays(fn))
+
+
+def to_ivy_dtype(dtype):
+    if not dtype or isinstance(dtype, str):
+        return dtype
+    if dtype in (int, float, bool) or ivy.is_native_dtype(dtype):
+        return ivy.as_ivy_dtype(dtype)
```

Comment:
Add 
`if isinstance(dtype, np_frontend.dtype):
       return dtype.ivy_dtype`

Reply:
Done. it was `_ivy_dtype` . Thanks for this and It works as expected now.